### ISSUE TITLE
Help using AKAs / languages

### ISSUE THREAD

**Issue by Chris230291** [cinemagoer/cinemagoer#405]
#### Issue description

Can someone help me figure out where I am going wrong please?
I am trying to translate titles.

#### Version of Cinemagoer, Python and OS

```
Name: cinemagoer
Version: 2022.2.11
Summary: Python package to access the IMDb's database
Home-page: https://cinemagoer.github.io/
Author: Davide Alberani
Author-email: da@mimante.net
License: GPL
Location: c:\users\chris\appdata\local\packages\pythonsoftwarefoundation.python.3.9_qbz5n2kfra8p0\localcache\local-packages\python39\site-packages
Requires: lxml, SQLAlchemy
Required-by:
```

#### Steps to reproduce the issue

```
import imdb
from imdb import helpers

ia = imdb.Cinemagoer()

movies = ia.search_movie('Tuż przed tragedią')
print(helpers.akasLanguages(movies[0]))
```

#### What's the expected result?

I expect to see all of the languages and titles shown here:
https://www.imdb.com/title/tt0462133/releaseinfo

#### What's the actual result?

Outputs:
`[(None, 'Tuż przed tragedią')]`

#### Additional details

Perhaps I am using it wrong?
I want to input a title in whatever language, find the movie/show, then see if the desired output language is in the AKAs.

The example given above is the Polish name of a show. Lets assume I want to translate it to English. I would get `Seconds from Disaster`

Thanks,
Chris

**Comment by daomengRen** [comment]
```
ia = Cinemagoer()
result = ia.get_movie('0111161')
print(result)
print(len(akasLanguages(result)))
```
https://www.imdb.com/title/tt0111161/releaseinfo
It doesn't return all akas, only 5, actually 79

**Comment by leodrivera** [comment]
It's because imdb have changed the layout of the site. If you look at its previus version, you'll see that it shows the list "exploded":
https://web.archive.org/web/20221120191839/https://www.imdb.com/title/tt0133093/releaseinfo?ref_=tt_dt_aka

In the current version, it shows the list collapsed:
https://www.imdb.com/title/tt0133093/releaseinfo/?ref_=tt_dt_aka

It also applies to other infos, like "Release Dates"

@alberanid 
Is it simple to fix it that so that it "explodes" the list before retrieving it?

**Comment by alberanid** [comment]
> Is it simple to fix it that so that it "explodes" the list before retrieving it?

It's almost impossible, being an async query.

Other pages had the same fate, but for the moment some old pages are still available with the old "complete" format. Something like "fullcredits" (e.g. https://www.imdb.com/name/nm0000206/fullcredits ) per people.

I doubt that a similar page exists for "release dates", but if someone find it, let us know.

**Comment by drzraf** [comment]
Possibly dup' of #390

**Comment by alberanid** [comment]
This issue is no longer relevant due to issue #557: ss of June 2026, Cinemagoer no longer analyzes web pages and can only rely on non-commercial datasets.
